assignments/function-conversion.js

```diff
@@ -1,21 +1,21 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
-// let myFunction = function () {
+// let myFunction = () => {
 // console.log("Function was invoked!");
 // };
 // myFunction();
 
-// let anotherFunction = function (param) {
+// let anotherFunction = (param) => {
```

Comment:
These arrow functions look great! Uncomment and run them next time to make sure they're working